go/heif/heif.go

```diff
@@ -321,7 +321,12 @@ func (c *Context) ReadFromFile(filename string) error {
 
 func (c *Context) ReadFromMemory(data []byte) error {
 	// TODO: Use reader API internally.
-	err := C.heif_context_read_from_memory(c.context, unsafe.Pointer(&data[0]), C.size_t(len(data)), nil)
+	cData := C.CBytes(data)
```

Comment:
Why not store a reference to `data` in the `Context`? This would make sure it is kept alive until the `Context` is released.

If you do that, make sure to clear the referene manually if it is no longer needed (e.g. when loading from a file in the same context).